edge-modules/load-gen/Program.cs

```diff
@@ -30,42 +32,45 @@ static async Task Main()
                     ModuleUtil.DefaultTransientRetryStrategy,
                     Logger);
 
-                using (var timers = new Timers())
```

Comment:
are you going to remove Timers class since no usage anymore?

Reply:
Yes I can do this if we don't think they will be necessary for anything else in the future

Reply:
please remove it.